README.md

```diff
@@ -56,14 +57,36 @@ import loki from 'k6/x/loki';
 let conf = loki.Config("localhost:3100");
 ```
 
+### Class `Labels(labels)`
+
+The class `Labels` allows the definition of custom labels that can be used
+instead of the built-in labels. An instance of this class can be passed as
+fifth argument to the `loki.Config` constructor.
+
+| argument | type   | description | default |
+| -------- | ------ | ----------- | ------- |
+| labels   | object | An object of type string (label name) to list of strings (possibel label values). | - |
+
+**Example:**
+
+```js
+import loki from 'k6/x/loki';
+let labels = loki.Labels({
+  "format": ["json", "logfmt"], // must contain at least one of the supported log formats
```

Comment:
Is `format` a special  kind of label that is send to loki either way and it know what to do with it?

Reply:
Yes, it is. The label defines the format of the log line.
Because of this, we can make sure, that if we filter on that label when querying, the log lines can be parsed.